radix-transactions/src/manifest/parser.rs

```diff
@@ -282,134 +308,44 @@ pub enum SborValueIdent {
     NonFungibleLocalId,
     AddressReservation,
     NamedAddress,
+    Intent,
+    NamedIntent,
 }
 
-impl SborValueIdent {
+impl ManifestValueIdent {
     pub fn from_ident(ident: &str) -> Option<Self> {
         let value = match ident {
             // ==============
             // SBOR composite value types
             // ==============
-            "Enum" => SborValueIdent::Enum,
-            "Array" => SborValueIdent::Array,
-            "Tuple" => SborValueIdent::Tuple,
-            "Map" => SborValueIdent::Map,
+            "Enum" => ManifestValueIdent::Enum,
```

Comment:
Not sure, but maybe we should use `Self` here and below to spare some work when renaming in the future?